modules/ROOT/pages/design-assigning-permissions.adoc

```diff
@@ -55,6 +55,33 @@ To share your projects and add the corresponding permissions directly from API D
 
 NOTE: This action sends the user an email informing a project has been shared with them.
 
+== Changing Ownership of Projects from Design Center
```

Comment:
We should also add that the option to change a project's owner is only available for organization administrators, or the current project owner.